<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc/MainWindow.qrc">
    <normaloff>:/rsc/icons/MainWindowIcon.png</normaloff>:/rsc/icons/MainWindowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Database Information</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="databaseTree">
          <column>
           <property name="text">
            <string>File Path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Function Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Function Line</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Views</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="viewsTree"/>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="viewsTab"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Database"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionSave_As_Database"/>
    <addaction name="actionSave_Database"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Analyze"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionStart_Analyze">
   <property name="text">
    <string>Start Analyze</string>
   </property>
  </action>
  <action name="actionVisual_Studio_Project">
   <property name="text">
    <string>Visual Studio Project</string>
   </property>
  </action>
  <action name="actionDatabase_2">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCreate_Database">
   <property name="text">
    <string>Create Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionSave_As_Database">
   <property name="text">
    <string>Save As Database</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rsc/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
